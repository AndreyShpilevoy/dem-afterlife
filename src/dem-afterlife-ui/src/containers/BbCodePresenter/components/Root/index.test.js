/* eslint no-undef: 0, fp/no-unused-expression: 0, fp/no-nil: 0, fp/no-mutation:0 , react/jsx-filename-extension:0 */

import React from "react";
import { shallow } from "enzyme";

import { RootPure } from "./index";

const classes = {
  root: "default-Root-root"
};

describe("RootPure", () => {
  it("component should match expected snapshot", () => {
    expect(
      shallow(<RootPure classes={classes}>content</RootPure>)
    ).toMatchSnapshot();
  });
});
