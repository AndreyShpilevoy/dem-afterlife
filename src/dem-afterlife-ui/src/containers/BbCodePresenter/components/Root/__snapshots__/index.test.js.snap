// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootPure component should match expected snapshot 1`] = `
<span
  className="default-Root-root"
>
  content
</span>
`;
